contracts/test/protocore/TestProtocore.sol

```diff
@@ -100,6 +100,15 @@ contract TestProtocore is Protocore {
         fvsCounts[_height] = fvsCounts[_height].add(1);
     }
 
+    function inValidatorSet(address, uint256)
+        public
+        view
+        returns (bool)
+    {
+        // @todo (pro): Implement it.
```

Comment:
Todo?

Reply:
I've removed todo. It's sufficient to return true from the function currently as tests are not using this one (it's only needed to do negative testing).